tests/spread/integration/unzip/task.yaml

```
@@ -0,0 +1,15 @@
+summary: Integration tests for unzip
+
+execute: |
+  rootfs="$(install-slices base-files_bin dash_bins grep_bins grep_deprecated \
+    sed_bins unzip_bins unzip_scripts zip_bins)"
```

Once we include the `grep`, `sed` and `dash` slices in `unzip_scripts`, we can reduce the above to the following:

```suggestion
  rootfs="$(install-slices base-files_bin unzip_bins unzip_scripts zip_bins)"
```